components/Header/Header.js

```diff
@@ -0,0 +1,19 @@
+import styles from './header.module.scss'
+import Link from 'next/dist/client/link'
+
+const Header = () => {
+  return (
+    <header className={styles.container}>
```

Comment:
<header className={styles.container}> ??? i dont get why its styles.container?

should it be styles.header?

Reply:
ok